py/objgenerator.c

```diff
@@ -230,6 +230,19 @@ STATIC const mp_rom_map_elem_t gen_instance_locals_dict_table[] = {
 
 STATIC MP_DEFINE_CONST_DICT(gen_instance_locals_dict, gen_instance_locals_dict_table);
 
+#if MICROPY_PY_GENERATOR_ATTRS
+STATIC void gen_attr(mp_obj_t self_in, qstr attr, mp_obj_t *dest) {
+	mp_obj_gen_instance_t *self = MP_OBJ_TO_PTR(self_in);
```

Comment:
Incorrect indentation here and everywhere else, please see https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md